GOVERNANCE.md

```diff
@@ -25,7 +25,7 @@ This project includes the following roles.
 
 **3.2. Balance**. The development process should balance the interests of all stakeholders.
 
-**3.3. Harmonization**. Good faith efforts shall be made to resolve any conflicts.
+**3.3. Harmonization**. Good-faith efforts shall be made to resolve any conflicts.
 
 ## 4. Trademarks
 
```

Comment:
As this is based on someone else document I left it as it was, but I've no objections to that improvements.